src/horizontalaligner.cpp

```diff
@@ -516,15 +516,15 @@ void Alignment::Reset()
     Object::Reset();
 
     m_xRel = 0;
-    m_time = Fraction(0, 1);
+    m_time = Fraction(0);
     m_type = ALIGNMENT_DEFAULT;
 
-    ClearGraceAligners();
+    this->ClearGraceAligners();
 }
 
 Alignment::~Alignment()
 {
-    ClearGraceAligners();
+    this->ClearGraceAligners();
```

Comment:
Is there a reason for adding `this->` to member-function calls? (Here and elsewhere).

Reply:
Style according to the [coding guidelines](https://book.verovio.org/contributing/guidelines.html#class-method-and-member-names).

Reply:
> <img width="718" alt="Screenshot 2025-01-25 at 13 04 31" src="https://github.com/user-attachments/assets/6262f05a-9346-4943-b4ee-068f0f404983" />

I have not been doing this (pun intended), but I will start doing it.   This is perhaps a later addition to the style guide due to the last sentence.

Reply:
It was added to the style guide after [this discussion](#2147 ) (pun intended) which I started after diving into the codebase. After using the convention for quite some time, member function calls without `this->` start looking odd.